### ISSUE TITLE
optionally log HTTP metrics headers in gRPC and Gateway servers

### ISSUE THREAD

**Issue by dougfort** [DecipherNow/gm-fabric-go#196]
Add a config option (in templates) to specify logging HTTP metrics headers . When we start using Envoy and/or gm-fabric-proxy we are going to be interested in what's coming in.

**Comment by lucasmoten** [comment]
If something like this is implemented, it would be nice if there was configuration of which headers to log, to avoid getting overly verbose if something sends in hundreds of headers

**Comment by dougfort** [comment]
Yes. I was thinking along the lines of ```log-http-headers=xxx``` where ```xxx``` is a prefix, or ```all```. But I think you are right that we want to be more sophisticated.
